### PR TITLE
Refactoring tests

### PR DESCRIPTION
**Pull request by raeeschachar**

### CODE REVIEW

**Comment by raeeschachar** [comment]
@edx/testeng 
Last part of [ticket](https://openedx.atlassian.net/browse/TE-1941). Please review.

File: regression/tests/studio/test_studio_textbooks.py [hunk 1]
```diff
@@ -36,40 +36,34 @@ def setUp(self):
...
         )
         self.textbook_page.upload_textbook('test_pdf.pdf')
-        self.textbook_page.click_textbook_submit_button()
 
-    def test_textbook_verify_on_lms(self):
-        """
-        Verifies that user can add and visit textbook on LMS
-        """
-        self.add_textbook()
+        # Verify the added textbook on LMS
```
Comment (jzoldak) [inline regression/tests/studio/test_studio_textbooks.py]: nit: this comment should be 1 line down, after the "click_textbook_submit_button()" line.

**Comment by raeeschachar** [comment]
@jzoldak 
Done. Please review.

**Comment by raeeschachar** [comment]
jenkins run e2e

**Comment by raeeschachar** [comment]
jenkins run e2e

**Comment by jzoldak** [comment]
Looks good to merge. 👍

**Pull request merged**